lib/file-info-view.coffee

```diff
@@ -10,10 +12,31 @@ class FileInfoView extends HTMLElement
     @bufferModified.classList.add('buffer-modified')
     @appendChild(@bufferModified)
 
+    @newTooltip()
+
     @activeItemSubscription = atom.workspace.onDidChangeActivePaneItem =>
       @subscribeToActiveItem()
     @subscribeToActiveItem()
 
+    clickHandler = =>
+      text = @getActiveItem()?.getPath?() or @getActiveItem()?.getTitle?() or ''
+      atom.clipboard.write(text)
+      setTimeout =>
+        @newTooltip()
+      , 2000
+
+    @currentPath.addEventListener('click', clickHandler)
+    @clickSubscription = new Disposable => @removeEventListener('click', clickHandler)
+
+  newTooltip: ->
```

Comment:
What I've seen (and used) most often across Atom packages is something like `handleX` when binding stuff to events, so `handleCopiedTooltip` would also work here.